packages/ui/react-ui/src/components/ScrollArea/ScrollArea.stories.tsx

```
@@ -114,34 +125,54 @@ export const NestedScrollAreas = {
-              className='shrink-0 bs-full is-[16rem] grid grid-rows-[min-content_1fr_min-content] overflow-hidden border border-separator'
+              className='shrink-0 bs-full is-[16rem] grid grid-rows-[min-content_1fr_min-content] border border-separator'
             >
-              <div className='flex shrink-0 p-2 border-be border-separator'>Column {column.id}</div>
-              <ScrollArea thin orientation='vertical'>
-                <div className='flex flex-col p-3 gap-1'>
+              <header className='flex shrink-0 p-2 border-be border-separator'>Column {column.id}</header>
+              <ScrollArea.Root thin orientation='vertical'>
+                <ScrollArea.Viewport classNames='plb-2 pli-2 gap-2'>
                   {Array.from({ length: column.count }, (_, i) => (
-                    <div key={i} className={`p-2 border border-separator rounded-sm text-sm`}>
+                    <div key={i} role='listitem' className={`shrink-0 p-2 text-sm border border-separator rounded-sm`}>
```

Biome (2.3.14)

[error] The elements with this role can be changed to the following elements:
<li>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)